src/coord/src/catalog.rs

```diff
@@ -1837,7 +1836,7 @@ impl Catalog {
             catalog: &Catalog,
             id: GlobalId,
             indexes: &mut Vec<GlobalId>,
-            complete: &mut bool,
+            sources: &mut Vec<GlobalId>,
```

Comment:
Can this be renamed to `unmaterialized` or something more specific? Same below.

Reply:
done